WORKSPACE

```diff
@@ -29,6 +29,14 @@ rules_java_dependencies()
 
 rules_java_toolchains()
 
+http_archive(
+    name = "zlib",
+    build_file = "@com_google_protobuf//:third_party/zlib.BUILD",
```

Comment:
It is unusual to include a BUILD file in the `http_archive`s. Is this required for this dependency to work?

Reply:
I believe this might be happening because you have not added the dependency to `third_party/versions.bzl`.

Please do that, including adding the zlib version to `HTTP_DEPENDENCY_VERSIONS`, and then follow the instructions in:
https://github.com/oppia/oppia-android/wiki/Updating-Maven-Dependencies

Reply:
Sounds good.